alacritty/src/config/font.rs

```diff
@@ -22,6 +22,11 @@ pub struct Font {
     /// Glyph offset within character cell.
     pub glyph_offset: Delta<i8>,
 
+    #[config(deprecated = "To enable thin strokes on macOS, run:\n$ defaults write <domain> \
+                           AppleFontSmoothing -int 0\nwhere <domain> is -g (for system-wide) or \
+                           io.alacritty (for just Alacritty).\nThen reboot your system or \
+                           relaunch Alacritty for the changes to take effect.\nTo disable thin \
+                           strokes, run:\n$ defaults delete <domain> AppleFontSmoothing")]
```

Comment:
This is overly verbose and I'm pretty sure we should shorten it. It's also not a deprecated setting, but entirely removed one.

I'd suggest to alter serialization and have a `removed` option (look for "deprecated" in `alacritty_config_derive`.)

The resulted message should be more like
```rust
#config(removed = "The behavior of font smoothing is now controlled with AppleFontSmoothing user default.")]
```

Reply:
Sure; I will trim it a bit and add a `removed` attribute.

Reply:
I'd still like to put the command users need to run in the warning. Not everyone knows what a "user default" is.

Reply:
Here is take 2.
<img width="800" alt="use_thin_strokes removal warning" src="https://user-images.githubusercontent.com/2592855/178159403-a65c7e53-be93-45b0-b351-f9ab2dc45b7e.png">